.github/workflows/ci.yaml

```diff
@@ -1,9 +1,6 @@
 name: CI
 
 on:
-  push:
-    branches:
-      - '*'
```

Comment:
I would just list the main branch here to ensure that master isn't borked when it is updated.